src/inc/msquic.ver

```diff
@@ -3,21 +3,27 @@
 //    Licensed under the MIT License.
 //
 
+#define STR_HELPER(x) #x
+#define STR(x) STR_HELPER(x)
+
+#ifndef VER_BUILD_ID
+#define VER_BUILD_ID 0
+#endif
+
+#ifndef VER_SUFFIX
+#define VER_SUFFIX "-private"
+#endif
+
 #define VER_COMPANYNAME_STR         "Microsoft Corporation"
 #define VER_FILEDESCRIPTION_STR     "Microsoft\256 QUIC Library"
 #define VER_INTERNALNAME_STR        "msquic"
 #define VER_LEGALCOPYRIGHT_STR      "\251 Microsoft Corporation."
 #define VER_PRODUCTNAME_STR         "Microsoft\256 QUIC"
-
-#define VER_FILEVERSION             1,0,0,0
-#define VER_FILEVERSION_STR         "1.0.0\0"
-
-#define VER_PRODUCTVERSION          1,0,0,0
-#define VER_PRODUCTVERSION_STR      "1.0.0\0"
+#define VER_PRODUCTVERSION_STR      "1.0.0." STR(VER_BUILD_ID) VER_SUFFIX "\0"
```

Comment:
Is changing this going to break the version update and packaging scripts?

Reply:
Perhaps. I'll look into it.

Reply:
Fixed the update version script. I think that should be it.

Reply:
create-release.ps1 looks to need the same line changed I think.